### ISSUE TITLE
Content support on CPAP supply reordering

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#62183]
## Issue Description
Supply reordering team opened [an intake](https://app.zenhub.com/workspaces/sitewide-content-accessibility-and-ia-63a1d63232beba0011a7833f/issues/gh/department-of-veterans-affairs/va.gov-team/61624) to add CPAP supplies to the existing hearing aid batteries tool. 

We'll need to provide content support for adjusting H1s and other content in the tool. 

---
## Tasks
- [ ] Confirm IA recs before starting content
- [ ] Suggest H1 and other content changes to online tool and related entry point pages

## Acceptance Criteria
- [ ] Content recs provided to product team in intake ticket

**Comment by laurwill** [comment]
Hi @d-beyer ,
As we just chatted, could you start a draft for this new CPAP supply ordering page? I've updated the top of this ticket with links and instructions, but let me know when you have questions. We'll aim to have questions to send to the SMEs on Monday. Thank you!

**Comment by laurwill** [comment]
Hi @d-beyer , I left you a few more comments in the google doc — after one more pass, I think we'll be ready to move this into a Word doc and send it over to SMEs. Thanks for all your work on this!

**Comment by d-beyer** [comment]
@laurwill I did my pass! Let me know if there's anything else I can do.

**Comment by d-beyer** [comment]
@laurwill The draft is attached, both as a version that maintains our comments and a clean copy.

[Draft_Order_CPAP_supplies_no_comments.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/12490438/Draft_Order_CPAP_supplies_no_comments.docx)
[Draft_Order_CPAP_supplies_with_comments.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/12490439/Draft_Order_CPAP_supplies_with_comments.docx)

**Comment by laurwill** [comment]
Hi @d-beyer , while we wait for answers from SMEs, you start building this new CPAP page in Drupal. Here's the [Sharepoint document](https://dvagov-my.sharepoint.com/:w:/g/personal/laura_willwerth_va_gov/EWWA8N14uW5AhhWdKSAfx9wBht4gc2ztk_RnTllOK5ssDA?e=jH5jjo) with the latest version. We can connect on Slack about how to set up a brand new page.

**Comment by d-beyer** [comment]
Hi @laurwill, I built the [draft of the CPAP page in drupal](http://preview-prod.vfs.va.gov/preview?nodeld=61493).

**Comment by laurwill** [comment]
Thanks so much @d-beyer ! This looked great. You can look at the revision history in Drupal to see my changes, but here's a quick list:
- Fixed the TTY aria codes (it should be on the TTY: as well as the 711)
- Adjusted 2 more bulleted list intros to be full sentences
- Adjusted the phone help options to drop the DLC acronym (seems unnecessary to introduce) and say "we're here" instead of "customer service representatives are available"
- Changed a couple instances of "items" and "accessories" to "supplies" for consistency

I'm going to collect our remaining SME questions along with the Drupal preview and pass to Jasmine to send to SMEs. Thanks for all your work on this!

**Comment by laurwill** [comment]
Oh and @megzehn (tagging you because Chelsea is out Monday), would you have time to copyedit this page early next week? Product team wants to bring this to staging review asap. I'll check slack Monday in case you have Qs (I'm also OOO). And if you can't get to it just let me know and we'll make another plan. Thanks! 

cc @strelichl for awareness

**Comment by megzehn** [comment]
@laurwill confirming I published this page and updated the left nav.

**Comment by laurwill** [comment]
@strelichl we can close this one out, page is live.
